src/app/shared/components/search/search.component.ts

```diff
@@ -1,15 +1,23 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+import { NgForm } from '@angular/forms';
 
 @Component({
   selector: 'app-search',
   templateUrl: './search.component.html',
-  styleUrls: ['./search.component.scss']
+  styleUrls: ['./search.component.scss'],
 })
 export class SearchComponent implements OnInit {
+  searchBtn = 'Search';
+  search = '';
 
-  constructor() { }
+  @Output() onSearch = new EventEmitter<string>();
 
-  ngOnInit(): void {
-  }
+  constructor() {}
+
+  ngOnInit(): void {}
 
+  onSubmit(searchForm: NgForm) {
+    this.onSearch.emit(searchForm.form.value.search);
+    this.search = '';
```

Comment:
`
onSubmit() {
    this.onSearch.emit(this.search);
    this.search = '';
}
`